styledComponents.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'styled-components/native';
```

Comment:
unnecessary, already exist a lib for the types of styled, the error it throws when you use it should told you wich lib is